1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -7,14 +7,14 @@
 
 */
 
-let people = [
+const people = [
   { name: "Bob", age: 30 },
   { name: "John", age: 20 },
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
-  /* todo */
+const peopleNames = people.map((e) => e.name);
```

Comment:
Just a minor nitpick: remember to use meaningful variable names, e.g.:
```suggestion
const peopleNames = people.map((person) => person.name);
```